contracts/hardhat.config.ts

```diff
@@ -52,6 +52,8 @@ function createConfig(network: string) {
   //       they are used to interact with L1 contracts in L2 deploy scripts
   if (network === "specularLocalDev") {
     config.companionNetworks = { l1: "localhost" };
+    config.gas = 2100000;
```

Comment:
why this is needed?

Reply:
without it, upgrading a proxy through the HH plugin fails with `code: 'UNPREDICTABLE_GAS_LIMIT'`

Reply:
fixed, this is no longer required